app/models/pglogical_subscription.rb

```diff
@@ -229,6 +230,12 @@ def assert_valid_schemas!
     end
   end
 
+  def assert_differnet_region!
```

Comment:
typo. "differnet" should read "different"

Reply:
😢 I'll fix that. Thanks for finding it!

Reply:
Maybe he's diff'ing internet regions.  :trollface: